chawkimbarki/Awesome-Books
# Awesome books with es6

# Refactor code to use ES6 and Update README 🚀

## Description 📝

This pull request updates the `README.md` file to match the project details and refactors the code to use ES6 features, such as arrow functions and modules.

## Changes 🛠️

- Updated README with project title, description, and instructions ✏️
- Set Up linter for `HTML`, `CSS`, and `JS`
- Copied the code from the previous project 📋
- Refactored the code to use ES6 syntax and features 💯
- Use Luxon to show time ⌚ 
- Turned all functions into arrow functions ➡️
- Divided the script into modules for better organization and readability 🗂️

## How to test 🧪

- Clone the repository and switch to the feature branch 🌱
- Open the `Index.html` to launch the project 🚀
- Check the `README.md` file and the code for any errors or typos ❌
- Provide feedback or approve the changes 👍
